executing/executing.py

```diff
@@ -257,30 +281,32 @@ def for_filename(
         module_globals=None,
         use_cache=True,  # noqa no longer used
     ):
+        # type: (Union[str, Path], Optional[Dict[str, Any]], bool) -> Any
```

Comment:
```suggestion
        # type: (Union[str, Path], Optional[Dict[str, Any]], bool) -> "Source"
```

(I'm assuming that annotating these more correctly as 'return an instance of cls' isn't really doable)